extensions/2.0/Khronos/KHR_avatar_skeleton_mapping/README.md

```diff
@@ -0,0 +1,148 @@
+# KHR_avatar_skeleton_mapping
+
+## Contributors
+
+- Ken Jakubzak, Meta
+- Hideaki Eguchi / VirtualCast, Inc.
+- K. S. Ernest (iFire) Lee, Individual Contributor / https://github.com/fire
+- Shinnosuke Iwaki / VirtualCast, Inc.
+- 0b5vr / pixiv Inc.
+- Leonard Daly, Individual Contributor
+- Nick Burkard, Meta
+
+## Status
+
+**Draft** – This extension is not yet ratified by the Khronos Group and is subject to change.
+
+## Dependencies
+
+Written against the glTF 2.0 specification.  
+Dependent on: `KHR_avatar`
+Typically used in conjunction with: `KHR_avatar_skeleton_biped`
+
+## Overview
+
+The `KHR_avatar_skeleton_mapping` extension provides a mechanism to map a skeleton rig to a reference rig, enabling retargeting and compatibility across different skeleton topologies. This extension is particularly useful for normalizing diverse rig structures across platforms and authoring tools. It provides one-to-one and one-to-many mapping; with the expectation that distributed weights add up to 1.0.
+
+## Extension Schema
+
+```json
+{
+  "extensions": {
+    "KHR_avatar_skeleton_mapping": {
+      "mappings": {
+        "Spine1": [
+          { "target": "spine", "weight": 0.5 },
+          { "target": "chest", "weight": 0.5 }
+        ],
+        "JawJoint": [{ "target": "jaw", "weight": 1.0 }]
+      }
+    }
+  }
+}
+```
+
+### Properties
+
+| Property   | Type   | Description                                                                |
+| ---------- | ------ | -------------------------------------------------------------------------- |
+| `mappings` | object | Dictionary mapping source joint names to reference rig joint names.        |
+| `target`   | string | Name of the reference joint.                                               |
```

Comment:
Assuming that you can include multiple instances of the KHR_avatar_skeleton_mapping extension, might it be worth calling that out explicitly, perhaps with an example?

If you can't include multiple instances of the KHR_avatar_skeleton_mapping extension, maybe it would make sense for `mappings` to be a Dictionary that maps from the name of a reference rig to the current dict that maps joints?

(a "maybe just me" nitpick: `mappings` in the plural here seems slightly odd to me, because it's a single skeleton mapping, unless it's referring to multiple joint mappings.)